### PR TITLE
Tiny PR - Tiny Improve

### PR DESCRIPTION
**Pull request by gastonmorixe**
- [x] Fix HTML (closes body and html tags :)
- [x] Add SEO meta-tags
- [x] Improve readability (max line length 80 chars)
- [x] Mobile support (add meta tags for responsive)
- [x] (Optional) Switched to [Fira Code](https://github.com/tonsky/FiraCode) font

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,137 +1,234 @@
...
-
-Invest with your PRs.
-
-<br/><br/><br/>
-
-<em>What's the goal of the tiny corp?</em><br/></br>
-
-To accelerate. We will commoditize the petaflop and enable AI for everyone.
+<head>
+  <meta charset="UTF-8">
```
Comment (gastonmorixe) [inline index.html:5]: added meta

File: index.html [hunk 2]
```diff
@@ -1,137 +1,234 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover" />
```
Comment (gastonmorixe) [inline index.html:6]: add mobile support

File: index.html [hunk 3]
```diff
@@ -1,137 +1,234 @@
...
+  <meta property="og:description"
+    content="Tiny Corp's tinygrad is an extremely simple and powerful neural network framework. We're developing the tinybox, and we're hiring. Join us today!">
+  <meta property="og:image" content="https://tinygrad.org/path_to_your_image.jpg">
+
+  <!-- Twitter -->
+  <!-- <meta property="twitter:card" content="summary_large_image"> -->
+  <meta property="twitter:url" content="https://tinygrad.org/">
+  <meta property="twitter:title" content="tinygrad: A simple and powerful neural network framework | Tiny Corp">
+  <meta property="twitter:description"
+    content="Tiny Corp's tinygrad is an extremely simple and powerful neural network framework. We're developing the tinybox, and we're hiring. Join us today!">
```
Comment (gastonmorixe) [inline index.html:29]: Add SEO Meta tags

File: index.html [hunk 4]
```diff
@@ -1,137 +1,234 @@
...
+  <!-- <meta property="twitter:image" content="https://tinygrad.org/todo.jpg"> -->
+
+  <!-- Canonical Tag -->
+  <link rel="canonical" href="https://tinygrad.org/">
+
+  <link rel="stylesheet" href="/assets/fira-code-6.2/fira_code.css">
```
Comment (gastonmorixe) [inline index.html:35]: Fira Code font

File: index.html [hunk 5]
```diff
@@ -1,137 +1,234 @@
...
+    }
+
+    h1 {
+      margin-top: 0;
+    }
+
+    main {
+      max-width: 80ch;
+      margin: 0 auto;
+    }
```
Comment (gastonmorixe) [inline index.html:62]: Improve legibility and center

File: index.html [hunk 6]
```diff
@@ -1,137 +1,234 @@
...
+
+    <br /><br /><br />
+
+    <em>What's the goal of the tiny corp?</em><br /></br>
+
+    To accelerate. We will commoditize the petaflop and enable AI for everyone.
+  </main>
+</body>
+
+</html>
```
Comment (gastonmorixe) [inline index.html:234]: Fix HTML, closed tags

**Comment by geohot** [comment]
So many files!

**Comment by gastonmorixe** [comment]
You can leave out the font and it’s just one file modified. You are welcome.

**Pull request closed without merging**